src/sdam/monitor.rs

```diff
@@ -18,7 +18,7 @@ use crate::{
     RUNTIME,
 };
 
-const DEFAULT_HEARTBEAT_FREQUENCY: Duration = Duration::from_secs(10);
+pub(super) const DEFAULT_HEARTBEAT_FREQUENCY: Duration = Duration::from_secs(10);
```

Comment:
We need to use the client's heartbeat frequency in the SRV polling implementation, and rather than having the monitor thread make the calculated value available back to the topology, it seemed simpler to just make this constant available to the other module and then redo the simply calculation.